lib/_tls_wrap.js

```diff
@@ -885,7 +885,8 @@ function Server(options, listener) {
     secureOptions: this.secureOptions,
     honorCipherOrder: this.honorCipherOrder,
     crl: this.crl,
-    sessionIdContext: this.sessionIdContext
+    sessionIdContext: this.sessionIdContext,
+    ticketKeys: this.ticketKeys
```

Comment:
Can you add a trailing comma?

Reply:
I certainly will! I didn't add one because there wasn't one to begin with 😅